Documentation/DocuBlocks/Rest/Cluster/put_cluster_maintenance.md

```diff
@@ -5,12 +5,21 @@
 @RESTHEADER{PUT /_admin/cluster/maintenance, Enable or disable the supervision maintenance mode}
 
 @RESTDESCRIPTION
-This API allows you to temporarily enable the supervision maintenance mode. Be aware that no
+This API allows to temporarily enable the supervision maintenance mode. Please be aware that no
 automatic failovers of any kind will take place while the maintenance mode is enabled.
-The _cluster_ supervision reactivates itself automatically _60 minutes_ after disabling it.
+The cluster supervision reactivates itself automatically at some point after disabling it.
 
-To enable the maintenance mode the request body must contain the string `"on"`. To disable it, send the string
-`"off"` (Please note it _must_ be lowercase as well as include the quotes).
+To enable the maintenance mode the request body must contain the string `"on"`
+(Please note it _must_ be lowercase as well as include the quotes). This will enable the
+maintenance mode for 60 minutes, i.e. the supervision maintenance will reactivate itself
+after 60 minutes.
+
+To enable the maintenance mode for a different duration than 60 minutes, it is possible to send
+the desired duration value (in seconds) as the request body. For example, sending `"7200"`
+(including the quotes) will enable the maintenance mode for 7200 seconds, i.e. 2 hours.
+
+To disable the maintenance mode the request body must contain the string `"off"` 
+(Please note it _must_ be lowercase as well as include the quotes).
```

Comment:
Let's add the versions in which the timing feature was introduced. I guess we are backporting till 3.7, but even if not, we should mention 3.8.3. Later minor versions from 3.9 on will also have it.

Reply:
Version information added to the docs.
No backport to 3.7 is planned for this, because 3.7 does not even have a UI for the maintenance.